lib/Dialog/Dialog.js

```diff
@@ -153,6 +165,12 @@ module.exports = kind(
 	*/
 	create: function () {
 		Popup.prototype.create.apply(this, arguments);
+		if(this.bodyComponents) {
+			this.$.message.destroy();
```

Comment:
Hi @rajyavardhanp, rather than create something we could need to destroy here, can we extend `initComponents` and conditionally create our components there instead?